### ISSUE TITLE
TensorFlow Output 

### ISSUE THREAD

**Issue by lpqdao** [iro-cp/FCRN-DepthPrediction#70]
After using the predict.py codes provided, I was able to obtain test images, however, there were black boxes on each side of the images as shown below. I was wondering why and how to fix this. Thanks.

![frame630](https://user-images.githubusercontent.com/37049175/42004832-82e55e0e-7a26-11e8-86d8-86949ab52789.PNG)

**Comment by nicolasrosa** [comment]
@lpqdao Hello Grace, I believe this is caused by the align_corners parameter from the tf.resize_images function.

**Comment by lpqdao** [comment]
Hello,

Are you talking about the problem regarding converting from 3-channel to 1-channel? Also which file is tf.resize_image in? Thanks.

Regards,
Grace Le

**Comment by nicolasrosa** [comment]
No, about the black boxes. In my code, it's in several places: train.py, validation.py and test.py.
